web/cypress/integration/route/batch-delete-route.spec.js

```diff
@@ -0,0 +1,165 @@
+    value0: 'value0',
+    label0_value0: 'label0:value0',
+  };
+
+  beforeEach(() => {
+    cy.login();
+  });
+
+  it('should successfully create 3 routes', function () {
+    cy.visit('/');
+    cy.contains('Route').click();
+    for (let i = 0; i < 3; i += 1) {
+      cy.contains('Create').click();
+      cy.contains('Next').click().click();
+      cy.get(selector.name).type(`test${i}`);
+      cy.get(selector.description).type(`desc${i}`);
+      cy.get(selector.hosts_0).type(data.host1);
+      cy.get(selector.uris_0).clear().type(`/get${i}`);
+
+      // config label
+      cy.contains('Manage').click();
+
+      // eslint-disable-next-line no-loop-func
+      cy.get(selector.drawerBody).within(() => {
+        cy.contains('button', 'Add')
+          .should('not.be.disabled')
+          .click()
+          .then(() => {
+            cy.get(selector.labels_0_labelKey).type(`label${i}`);
+            cy.get(selector.labels_0_labelValue).type(`value${i}`);
+            cy.contains('Confirm').click();
+          });
+      });
+
+      cy.contains('button', 'Next').should('not.be.disabled').click();
+      cy.get(selector.nodes_0_host).type(data.host2, {
+        timeout,
+      });
+      cy.get(selector.nodes_0_port).type(data.port);
+      cy.get(selector.nodes_0_weight).type(data.weight);
+      cy.contains('Next').click();
+      cy.contains('Next').click();
+      cy.contains('Submit').click();
+      cy.contains(data.submitSuccess);
+      cy.contains('Goto List').click();
+      cy.url().should('contains', 'routes/list');
+    }
+  });
+
+  it('should delete the route', function () {
+    cy.visit('/routes/list');
+    cy.contains(data.test0).get('[type="checkbox"]').check();
+    cy.contains(data.test2).get('[type="checkbox"]').check();
+    cy.contains('BatchDeletion Routes').should('be.visible').click({ timeout });
+    cy.get(selector.notification).should('contain', data.deleteRouteSuccess);
+    cy.get(selector.notificationClose).should('be.visible').click({
+      force: true,
+      multiple: true,
+    });
+  });
+
+  it('should batch delete the name of the route', function () {
+    cy.visit('/');
+    cy.contains('Route').click();
+    // full match
+    cy.get(selector.nameSearchInput).type(data.test0);
+    cy.contains('Search').click();
+    cy.contains(data.test1).should('not.exist');
+    cy.contains(data.test0).should('not.exist');
+    cy.contains(data.test2).should('not.exist');
+    // partial match
+    cy.get(selector.nameSearchInput).clear().type(data.test2);
+    cy.contains('Search').click();
+    cy.contains(data.test0).should('not.exist');
+    cy.contains(data.test1).should('not.exist');
+    cy.contains(data.test2).should('not.exist');
+    // no match
+    cy.get(selector.nameSearchInput).clear().type(data.testx);
+    cy.contains('Search').click();
+    cy.contains(data.test0).should('not.exist');
+    cy.contains(data.test1).should('not.exist');
+    cy.contains(data.test2).should('not.exist');
+  });
```

Comment:
Why do we need to test these search functions? We only need to judge that the data does not exist under https://github.com/apache/apisix-dashboard/pull/2502/files#diff-6918beb43c48e71616c503a8dffc6d9cc572f37a69d1e27f792b71fda2e5416aR117.